public/main.ts

```diff
@@ -265,7 +283,9 @@ function updateStats() {
 
 }
 function handleHarvest() {
-    player.orderHarvest();
+    player.farms.forEach((value: Farm, key: Crop) => {
+        value.playerHarvest(key);
+    });
```

Comment:
```suggestion
    player.farms.forEach(value => value.playerHarvest(key));
```

Remove one parameter and make a oneliner?